frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -17,10 +17,17 @@ import {
   deviceTypeDropdownItems,
 } from '@console/local-storage-operator-plugin/src/constants';
 import '../../attached-devices.scss';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { nodesDiscoveriesResource } from '../../../../../constants/resources';
 
 export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state, dispatch }) => {
   const { t } = useTranslation();
 
+  const [discoveriesData, discoveriesLoaded, discoveriesLoadError] = useK8sWatchResource<
```

Comment:
There cannot be discovery results without discovery, so we dont need to check for "discoveries data".